src/corehost/cli/hostpolicy.cpp

```diff
@@ -116,12 +110,30 @@ int run(const arguments_t& args)
         property_values.push_back(clrjit_path_cstr.data());
     }
 
+    bool set_app_paths = false;
+
+    // Runtime options config properties.
     for (int i = 0; i < g_init.cfg_keys.size(); ++i)
     {
+        // Provide opt-in compatible behavior by using the switch to set APP_PATHS
```

Comment:
It will be good to run scenarios like MusicStore to ensure that nothing is broken. What we want to confirm if that no one relies on this (if they do, they have a workaround), so that we do not need to have this opt-in.

I would expect that the default scenarios enabled by .NET Core do not enable this broken behaviour - is that correct?